src/Microsoft.AspNetCore.Server.Kestrel/Http/DateHeaderValueManager.cs

```diff
@@ -65,7 +64,7 @@ public DateHeaderValues GetDateHeaderValues()
 
             return new DateHeaderValues()
             {
-                Bytes = _activeDateBytes ? _dateBytes0 : _dateBytes1,
+                Bytes = _dateBytes,
```

Comment:
It should always read a value? For its use case (browser/proxy cache age) it doesn't matter if its stale for up to 2 seconds